### DISCUSSION TITLE
How to Import Excel files?

### DISCUSSION THREAD

**Discussion by lawilt919** [dbeaver/dbeaver#21769] [General]
Brand new to DBeaver. I use other applications for coding at work but I'm trying to use DBeaver tool to create a SQL test for interviewees.

I downloaded the application today but I can't figure out how to import an excel file.  When I go to File > Import, I can't find a way to get it to know I'm trying to load a local file to be able to query in the application.  Can someone help assist with the steps I'm missing?

Can the file be xlsx or does it need to be csv?

Thank you!

**Comment by lawilt919** [comment]
One additional question I thought of....Is there a way to set up DBeaver to use a specific type of SQL?

**Reply by LonwoLonwo** [reply]
What kind of specific? Could you please provide an example?

**Reply by lawilt919** [reply]
Yeah so I just want to use this for local files and not connect to any specific servers.  But I was curious if I could control it to specifically use SQL for SAP HANA Studio instead of MS SQL Server version or another version where the functions are slightly different.

**Reply by LonwoLonwo** [reply]
No, you could not. SQL depends on your connection. Without connection - our SQL Editor is just a notepad.

**Comment by LonwoLonwo** [comment]
Hello @lawilt919 

You can import data from XLSX https://github.com/dbeaver/dbeaver/wiki/Data-transfer#importing-data-from-excel-file
But only in PRO versions.

**Reply by lawilt919** [reply]
Hello @LonwoLonwo

Is there anyway in the free version to do something similar to create that table?  I'm trying to find a free version to do this with since it will just be to hold a couple of local files for these sample test questions.

**Reply by LonwoLonwo** [reply]
You can use Copy and then Advanced Paste (Ctrl+Shift+V) in the table.
